Configurations/unix-Makefile.tmpl

```diff
@@ -721,22 +690,19 @@ install_dev: install_runtime_libs
 		fn1=`basename "$$s1"`; \
 		fn2=`basename "$$s2"`; \
 		fn3=`basename "$$s3"`; \
-		: {- output_off(); output_on() unless windowsdll() or sharedaix(); "" -}; \
+		: {- output_off(); output_on() sharedaix(); "" -}; \
```

Comment:
Ok, so you decided to drop all windowsdll stuff...  That invalidates building for Mingw and possibly also for Cygwin.

-1

Reply:
Can you point to the lines in `Configurations/10-main.conf` that show that MingW and/or Cygwin actually use windows-style DLL's and APPLINK?

Reply:
Considering that especially Mingw essentially is GNU C toolchains for Windows (see http://mingw-w64.org/doku.php), that should be self evident enough.  Cygwin is more of a sub-system in its own right, so it's not quite as obvious.  Also, you have to know that applink and uplink are two halves of one whole, so when looking for applink, you also should look for uplink.

So:

https://github.com/openssl/openssl/blob/f4d8b29a26cc3ee0ef37e2073c192bdbb2b796c4/Configurations/10-main.conf#L1475-L1476

https://github.com/openssl/openssl/blob/f4d8b29a26cc3ee0ef37e2073c192bdbb2b796c4/Configurations/10-main.conf#L1488

https://github.com/openssl/openssl/blob/f4d8b29a26cc3ee0ef37e2073c192bdbb2b796c4/Configurations/10-main.conf#L1556-L1557

But most of all, you could look at how `windowsdll` is defined, in this very file:

https://github.com/openssl/openssl/blob/f4d8b29a26cc3ee0ef37e2073c192bdbb2b796c4/Configurations/unix-Makefile.tmpl#L11

Reply:
But, if you don't believe me, feel free to install msys2/mingw as well as cygwin on some Windows box and try things out for yourself